### DISCUSSION TITLE
pytorch tensor and inference

### DISCUSSION THREAD

**Discussion by swkms** [NVIDIA/VideoProcessingFramework#398] [Q&A]
Hello, I use VPF decoding and yolo model, nv12 ->yuv420 ->rgb24 ->rgb24Planar ->python tensor ->"run_inference" in the example. Can I use rgb24Planar to give self. bindings [0] without converting to python tensor

**Comment by contentis** [comment]
If you want to use the python interface, you need to use the tensor conversion as the binding is a raw byte stream that needs to be interpreted. This should not add any overhead and therefore is not an issue.
